service/debugger/debugger.go

```diff
@@ -291,6 +369,7 @@ func (d *Debugger) detach(kill bool) error {
 }
 
 var ErrCanNotRestart = errors.New("can not restart this target")
+var ErrNotRecording = errors.New("debugger is not recording")
```

Comment:
nit: Add doc comment to this (and above since we're here).

Reply:
Done.